edit-post/components/header/header-toolbar/index.js

```diff
@@ -34,7 +33,6 @@ function HeaderToolbar( { hasFixedToolbar, isLargeViewport } ) {
 			<EditorHistoryUndo />
 			<EditorHistoryRedo />
 			<TableOfContents />
-			<MultiBlocksSwitcher />
```

Comment:
I wonder if this `MultiBlocksSwitcher` is already handling showing the switcher only for multi selection.

We should:

- Delete this component since you duplicated its logic in `block-toolbar`
- Or Use it in `block-toolbar` directly